src/main.rs

```diff
@@ -187,15 +208,14 @@ fn main() {
 }
 
 #[cfg(feature = "debug_ctap")]
-fn print_packet_notice(notice_text: &str, timer: &Timer) {
-    let now = timer.get_current_clock().flex_unwrap();
-    let now_us = (Timestamp::<f64>::from_clock_value(now).ms() * 1000.0) as u64;
+fn print_packet_notice(notice_text: &str, clock: &LibtockClock) {
```

Comment:
We shouldn't use `LibtockClock` here. The `clock` module should expose a `Clock` (implementing `embedded_time::Clock`). The implementation would be controlled by a board selection feature (but for now it can be `LibtockClock`).

Reply:
I hided it in the clock module added. PTAL.